deployments/rialto/dashboard/prometheus/prometheus.yml

```diff
@@ -14,4 +14,4 @@ scrape_configs:
     scrape_interval: 15s
 
     static_configs:
-      - targets: ['relay-eth-exchange-sub:9616']
+      - targets: ['relay-eth-exchange-sub:9617']
```

Comment:
But relay node in `relay-eth-exchange-sub` node still uses port 9616 to expose its prometheus metrics. So iiuc prometheus container won't be able to fetch metrics from that relay.